File: AI2020OS/Models/AIServiceCatalogModel.m
//
//  AIServiceCatalogModel.m
//  AI2020OS
//
//  Created by admin on 15/8/24.
//  Copyright (c) 2015年 ___ASIAINFO___. All rights reserved.
//

#import "AIServiceCatalogModel.h"
#import "AINetEngine.h"

@implementation AIServiceCatalogModel



- (void)test
{
    
    AIMessage *message = [AIMessage message];
    
    
    NSDictionary *body = @{
                           @"data":@"data"
                           };
//    NSDictionary *header = @{};
//    
//    
//    [[AINetEngine defaultEngine] configureCommonHeaders:header];
    
    
    [message.body addEntriesFromDictionary:body];
    message.url = @"http://198.168.1.9/server";
    
    [[AINetEngine defaultEngine] postMessage:message success:^(NSDictionary *response) {
        //AIServiceCatalogModel *model = [[AIServiceCatalogModel alloc] initWithDictionary:response error:nil];
        
    } fail:^(AINetError error, NSString *errorDes) {
        
    }];
    
    
}




@end

@implementation AIQueryHotSearchResponse
@end

File: AI2020OS/Models/AIServiceCatalogModel.swift
//
//  ServiceCatalogModel.swift
//  AI2020OS
//
//  Created by admin on 15/5/29.
//  Copyright (c) 2015年 ___ASIAINFO___. All rights reserved.
//

import Foundation
import JSONJoy

//订单列表model
struct AICatalogListModel : JSONJoy {
    var catalogArray = Array<AICatalogItemModel>()
    init() {

    }
    
    init(_ decoder: JSONDecoder) {
        
        if var jsonArray = decoder["catalog_list"].array {
            catalogArray = Array<AICatalogItemModel>()
            for subDecoder in jsonArray {
                catalogArray.append(AICatalogItemModel(subDecoder))
            }
        }
    }
}

//订单列表项model
struct AICatalogItemModel : JSONJoy{

    var catalog_id : Int?
    var catalog_name : String?
    
    init(){
        
    }
    
    init(_ decoder: JSONDecoder) {
        var id = decoder["catalog_id"].integer
        catalog_id = decoder["catalog_id"].integer
        var name = decoder["catalog_name"].string
        catalog_name = decoder["catalog_name"].string
    }
}